tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
 envlist =
-	{python27,python34}-{django16,django17,django18}-{lpclient32,lpclient41},
-	python33-{django16,django17,django18}-{lpclient32,lpclient41},
-	python27-django14-lpclient32,
+	{py27,py34,py35}-{django18,django19,django110}-{lpclient41,lpclient5},
```

Comment:
Mind adding Django's master branch (https://github.com/MarkusH/django-dynamic-forms/blob/master/tox.ini#L25) with `ignore_outcome: true`: https://github.com/MarkusH/django-dynamic-forms/blob/master/tox.ini#L27-L28 ? This will then tell us about thinks that break / deprecate in future versions.